{
  "comments": [
    {
      "unresolved": false,
      "key": {
        "uuid": "a1de2ea1_24587961",
        "filename": "/PATCHSET_LEVEL",
        "patchSetId": 1
      },
      "lineNbr": 0,
      "author": {
        "id": 682
      },
      "writtenOn": "2024-01-29T16:25:21Z",
      "side": 1,
      "message": "Hmm I can\u0027t reproduce the problem in my local... Can it be related to the PATH setting used during the test ?",
      "revId": "1840db9b3d186eea133c0011603090c6c4a1ca74",
      "serverId": "270e2033-b340-4cff-9539-693957ebf0e7"
    }
  ],
  "submitRequirementResults": []
}